src/AppBundle/Manager/SubscriptionManager.php

```diff
@@ -93,14 +120,31 @@ public function saveSubscription(Subscription $subscription)
     {
         $this->em->persist($subscription);
         $this->em->flush($subscription);
+
+        $this->dispatcher->dispatch(
+            SubscriptionEvents::POST_WRITE,
+            new SubscriptionEvent($subscription->getId(), $subscription)
+        );
     }
 
     /**
      * @param Subscription $subscription
+     * @param bool $dispatch
      */
-    public function updateSubscription(Subscription $subscription)
-    {
+    public function updateSubscription(
+        Subscription $subscription,
+        $dispatch = true
```

Comment:
As mentioned earlier, a param whether or not to dispatch the events. How can a consumer decide if this should be true or false if it is not clear what can happen.